### DISCUSSION TITLE
FOLA feature concepts, sketches, and thinking

### DISCUSSION THREAD

**Discussion by fancyham** [hackforla/food-oasis#1550] [Ideas]
Let's try using this discussion thread to share concepts and ideas in a way that we can view, comment, and discuss them!

- (this is related to #1549)

**Comment by fancyham** [comment]
"Crazy 8" concept brainstorming exercise by design team on 2022-10-27

Process:
- Folded up paper into 8 segments.  Sketched for 15 minutes.  Posted sketches and talked about our ideas.  Then used stickers to vote. 

Popular ideas (3+ votes) were:
- Design system website as a repository of styles and code for design + dev
- Listing food delivery / meals on wheels-type services on site
- Generate printed listings / fliers from search results
- Listing other ways to get free food in results (SNAP, CalFresh, School lunches, etc.)
- Ability to send corrections or feedback about a food pantry / meal listing

![2022-10-27 - 'Crazy 8' Concept sketching](https://user-images.githubusercontent.com/3376957/217133472-cca5a501-27b0-4bdf-a6ab-6bb427fa5b51.jpg)

https://www.figma.com/file/D3oq9QOXl0rFkwrEUJbABs/FOLA-Design-%231?node-id=8714%3A16103&t=tUvyS8pPdeS9aDvZ-4

**Comment by narae-ux** [comment]
Brainstorming for new features (Dec 2022) 

Context: Bryan prompted me to write down some notes of what I’d like to see on the site before going too deep into the documentation, just to see what ideas might come up initially - aka without considering possible constraints.

[Figma link](https://www.figma.com/file/9hPmNSsyaH2VMGNz5mffmI/FOLA-Design-%232?node-id=1604%3A593&t=XEVmPeKhvoMCDXh6-0)

![Screen Shot 2023-02-16 at 12 41 12 PM](https://user-images.githubusercontent.com/86501066/219482113-cf994cad-f604-49d0-89ef-47a5cc0acab6.png)

![Screen Shot 2023-02-16 at 12 41 41 PM](https://user-images.githubusercontent.com/86501066/219482191-05f4336c-d3d6-4755-b29c-f9b663c186cc.png)
